src/utils/api/client.js

```diff
@@ -7,15 +7,28 @@ export class Client {
 
   axiosConfig = {
     timeout: 10000,
-  }
+  };
+
+  http = null;
+
+  rpc = ({ event, params, data }) =>
+    new Promise((resolve, reject) => {
+      if (this.socket.disconnected) {
+        reject(new Error('socket not connected'));
+        return;
+      }
+      this.socket.emit('request', { method: event, params: params || data || {} }, (response) => {
+        if (Object.keys(response).length && response.error) {
+          return reject(response);
+        }
+        return resolve(response);
+      });
+    });
 
-  http = null
+  rest = (config) => this.http?.request({ ...this.http.defaults, ...config });
 
   create({ rpc, rest }) {
```

Comment:
idk why, but you have this deprecated. Should be `{ws, http}` instead (according to the new service API schema)